lib/microservices_engine.rb

```diff
@@ -5,28 +5,29 @@
 
 module MicroservicesEngine
   class << self
+    # Setter method for the `build` portion of the engine, includes various validations
     def build=(b)
       @build = b if Rails.env.test? && b == '1.1.1'
 
-      # -- Semantic Versioning -- #
-      # - All version INCREASES are VALID
-      # - Version DECREASES are INVALID IF AND ONLY IF none of the more important
-      #   version numbers increase.
-      #   ~ That is to say, a major decrease is never valid.
-      #   ~ A minor decrease is only valid when the major version increases.
-      #   ~ A revision decrease is only valid when either the major or minor version increases.
+      # ---------------- Semantic Versioning ---------------- #
```

Comment:
FWIW, [there seems to be a gem for this](https://rubygems.org/gems/semantic/versions/1.4.1)